VEDA/utils.py

```diff
@@ -69,3 +69,34 @@ def get_config(yaml_config_file=DEFAULT_CONFIG_FILE_NAME):
         config_dict = yaml.load(config)
 
     return config_dict
+
+
+def scrub_url_info(url, params_to_scrub):
+    """
+    Scrub query params present in `params_to_scrub` from `url`
+
+    Arguments:
+        url (str): url
+        params_to_scrub (list): name of query params to be scrubbed from url
+
+    Returns:
+        url with query params scrubbed
+
+    >>> old_url = https://sandbox.veda.com/api/do?api_token=veda_api_key&job_name=12345&language=en&v=1
+    >>> new_url = https://sandbox.veda.com/api/do?v=1&job_name=12345&language=en&api_token=XXXXXXXXXXXX
+    """
+    parsed = urlparse.urlparse(url)
+
+    # query_params will be in the form of {'v': ['1'], 'job_name': ['12345'], 'api_token': ['cielo24_api_key']}
+    query_params = urlparse.parse_qs(parsed.query)
```

Comment:
we can use `urlparse.parse_qsl` to avoid query parameters being single valued lists.